ui/packages/catalog-ui-search/src/main/webapp/plugins/multi-select-actions.tsx

```diff
@@ -10,10 +9,5 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
- --}}
-<div class="content-left">
 
-</div>
-<div class="content-right">
-
-</div>
\ No newline at end of file
+export default (multiSelectButtons: any[]) => multiSelectButtons
```

Comment:
why is this necessary? Is it a type issue?

Reply:
What what you write here instead?

Reply:
I'm not really sure what this is used for, can it be deleted?

Reply:
The purpose is to make the multi-select button region extensible